assignments/arrays.js

```diff
@@ -63,32 +63,107 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// {"id":33,"car_make":"Jeep","car_model":"Wrangler","car_year":2011},
+console.log(`Car 33 is a ` + inventory[32].car_year + ` ` + inventory[32].car_make + ` ` + inventory[32].car_model );
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+// {"id":50,"car_make":"Lincoln","car_model":"Town Car","car_year":1999}];
+let lastCar = inventory.length-1;
+console.log(`Make and model of the last car is ` + inventory[lastCar].car_make +` and `+ inventory[lastCar].car_model);
```

Comment:
This is perfect!  Since you know its the last car, no need for any kind of loop.